src/mainBlock.css

```diff
@@ -0,0 +1,69 @@
+.stars{
+  display: flex;
+}
+
+.stars__star{
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star-active.svg");
+}
+
+.stars--0 .stars__star:nth-child(n) {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star.svg");
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star.svg");
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star.svg");
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin: 4px;
+  height: 16px;
+  width: 16px;
+  display: flex;
+  list-style: none;
+  background-image: url("images/star.svg");
+}
```

Comment:
The styles for `.stars--3 .stars__star:nth-child(n+4)` are almost identical to those in `.stars__star`. Consider removing duplicate properties to avoid redundancy.